### PR TITLE
Add z-index custom properties

### PR DESCRIPTION
**Pull request by hreyeslo**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -22,12 +23,15 @@
   --scrollbar-border-color: #999;
   --scrollbar-border-radius: 0;
   --scrollbar-box-shadow: none;
+  --scrollbar-hover-z-index: 30;
```
Comment (Julien-Marcou) [inline src/index.css]: Can you update it to `--scrollbar-z-index-hover` to make it consistent with other `hover` CSS properties ?

**Comment by hreyeslo** [comment]
Done! :slightly_smiling_face:

**Comment by Julien-Marcou** [comment]
I have some tests to run before releasing, you can expect the new version to be released within the next 24 hours

**Comment by hreyeslo** [comment]
No problem, I'm not in a hurry.

**Pull request merged**